polars/series/index.ts

```diff
@@ -222,6 +222,228 @@ export interface Series
    * Create a new Series that copies data from this Series without null values.
    */
   dropNulls(): Series;
+  /**
```

Comment:
moving this to `shared-trait` will reduce the code duplication between this and expr.

Reply:
I need to think about this, not sure how an interface going to help with 3 different function names.
If you have an example that would be great. Thx